lib/bing_ads_ruby_sdk/tasks/bing_ads_ruby_sdk.rake

```diff
@@ -1,4 +1,4 @@
-require "dotenv/load"
+require "dotenv/load" if Rails.env.development?
```

Comment:
@tdutreui-solocal Just in case. The SDK should be framework-agnostic. Not everyone will have Rails.